### PR TITLE
Id 1418/javascript tests

### PR DESCRIPTION
**Pull request by tylerchilds**

### CODE REVIEW

File: dist/js/styleguide.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
-$(function(){
```
Comment (abodnar-svitla) [inline dist/js/styleguide.js:1]: Why did you remove jQuery from here? You are using jQuery inside this module. This is possible that this module will be loaded and executed faster than jQuery. Should we add check that all libraries for this module are loaded and throw error if they are not?

File: dist/js/styleguide.js [hunk 2]
```diff
@@ -194,15 +193,17 @@ $(function(){
...
+})();
 
-$(function(){
+(function(){
 
   var Identity = window.Identity || {};
 
   Identity.forms = {};
-
+  var initialized = false;
```
Comment (abodnar-svitla) [inline dist/js/styleguide.js:203]: Why do we need this? If we don't want to initialize module twice we could check if Identity.forms.toggleShowPassword present.

File: test/fixtures/forms/password.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<label class="textfield textfield--label_bordered">
```
Comment (abodnar-svitla) [inline test/fixtures/forms/password.html:1]: Is it possible to have this template outside the test? If we change layout than test will still pass, because we have fixture in test.

**Review by abodnar-svitla — CHANGES_REQUESTED** [review]
Looks great. I added few thoughts in comments. Also is it possible to have better output in Circle? Right now it is not clear what test is passed and what is failed.

`Chrome 60.0.3112 (Linux 0.0.0): Executed 18 of 19 SUCCESS (0 secs / 0.272 secs)
LOG: <div class="notification-dismiss"></div>
Chrome 60.0.3112 (Linux 0.0.0): Executed 18 of 19 SUCCESS (0 secs / 0.272 secs)
Chrome 60.0.3112 (Linux 0.0.0): Executed 19 of 19 SUCCESS (0 secs / 0.38 secs)
Chrome 60.0.3112 (Linux 0.0.0): Executed 19 of 19 SUCCESS (0.394 secs / 0.38 secs)
Done in 2.25s.`

File: dist/js/styleguide.js (hunk 1 above)
Comment (tylerchilds) [reply dist/js/styleguide.js:1]: I was running into issues in testing, where the code in the `$(function(){})` wasn't executing. I can look into why.

File: dist/js/styleguide.js (hunk 2 above)
Comment (tylerchilds) [reply dist/js/styleguide.js:203]: That's a great idea, I'll update

File: test/fixtures/forms/password.html (hunk 3 above)
Comment (tylerchilds) [reply test/fixtures/forms/password.html:1]: Won't do for now, I'm over time for this and need to work on other things.

**Review by abodnar-svitla — APPROVED** [review]
